### ISSUE TITLE
LSP error while running under Windows 11

### ISSUE THREAD

**Issue by rudesome** [mrjosh/helm-ls#30]
Receiving the following error while trying to launch helm-ls in Windows 11:

```
[START][2023-05-08 11:27:49] LSP logging initiated
[ERROR][2023-05-08 11:27:49] .../vim/lsp/rpc.lua:734	"rpc"	"cmd.exe"	"stderr"	'{"level":"info","msg":"helm-lint-langserver: connections opened","time":"2023-05-08T11:27:49+02:00"}\n'
[ERROR][2023-05-08 11:27:49] .../vim/lsp/rpc.lua:734	"rpc"	"cmd.exe"	"stderr"	"panic: runtime error: slice bounds out of range [1:0]\n\ngoroutine 13 [running]:\ngithub.com/mrjosh/helm-ls/internal/util.URIToPath({0xc0002b0660?, 0xd82afe?})\n\t/home/runner/work/helm-ls/helm-ls/internal/util/strings.go:53 +0x145\n"
[ERROR][2023-05-08 11:27:49] .../vim/lsp/rpc.lua:734	"rpc"	"cmd.exe"	"stderr"	"github.com/mrjosh/helm-ls/internal/lsp.(*DocumentStore).normalizePath(0xc00074f020, {0xc0002b0660, 0x51})\n\t/home/runner/work/helm-ls/helm-ls/internal/lsp/document.go:61 +0x32\ngithub.com/mrjosh/helm-ls/internal/lsp.(*DocumentStore).DidOpen(0xc00074f020, {{{0xc0002b0660, 0x51}, {0xc000443cc8, 0x4}, 0x0, {0xc000287a40, 0x49}}})\n\t/home/runner/work/helm-ls/helm-ls/internal/lsp/document.go:33 +0x65\ngithub.com/mrjosh/helm-ls/internal/handler.(*langHandler).handleTextDocumentDidOpen(0xc0004394d0, {0x1fc3a78, 0xc0000340d8}, 0xc000796fc0, {0x14df9c65018, 0xc000796f90})\n\t/home/runner/work/helm-ls/helm-ls/internal/handler/handler.go:107 +0xb7\ngithub.com/mrjosh/helm-ls/internal/handler.(*langHandler).handle(0xc0004394d0, {0x1fc3a78, 0xc0000340d8}, 0xc000796fc0, {0x14df9c65018?, 0xc000796f90?})\n\t/home/runner/work/helm-ls/helm-ls/internal/handler/handler.go:49 +0x2d8\ngo.lsp.dev/jsonrpc2.ReplyHandler.func1({0x1fc3a78, 0xc0000340d8}, 0xc000707aa0, {0x14df9c65018?, 0xc000796f90?})\n\t/home/runner/go/pkg/mod/go.lsp.dev/jsonrpc2@v0.10.0/handler.go:35 +0xf6\ngo.lsp.dev/jsonrpc2.(*conn).run(0xc00045ea00, {0x1fc3a78, 0xc0000340d8}, 0xc0002b3720)\n\t/home/runner/go/pkg/mod/go.lsp.dev/jsonrpc2@v0.10.0/conn.go:206 +0x24b\ncreated by go.lsp.dev/jsonrpc2.(*conn).Go\n\t/home/runner/go/pkg/mod/go.lsp.dev/jsonrpc2@v0.10.0/conn.go:189 +0xb0\n"
```

any idea what this can be?

**Comment by qvalentin** [comment]
Seems to be a bug with the handling of filepaths on Windows. Must be fixed in the code.

**Comment by mrjosh** [comment]
What's the problem that causing this error?
Can we have a fix on this? @qvalentin @rudesome

**Comment by Anakael** [comment]
Same here on windows 10.

**Comment by qvalentin** [comment]
Fixed in [v0.0.10](https://github.com/mrjosh/helm-ls/releases/tag/)

**Comment by qvalentin** [comment]
Fixed in [v0.0.10](https://github.com/mrjosh/helm-ls/releases/tag/)
